docs/src/main/paradox/scala/http/compatibility-guidelines.md

```diff
@@ -71,13 +68,13 @@ Maven
     <dependency>
       <groupId>com.typesafe.akka</groupId>
       <artifactId>akka-http_$scala.binary_version$</artifactId>
-      <version>$akka25.version$</version>
+      <version>$project.version$</version>
```

Comment:
Oops, good catch. Did I recently introduce that?

Reply:
Looks like it (62ed17d15), might be a merge thing?